ResourceStaticAnalysis/src/ResourceStaticAnalysisCore/Engine/ResourceStaticAnalysisEngine.cs

```diff
@@ -72,18 +72,18 @@ public static ObjectCache<string> StringCache
         private readonly ManualResetEvent _coProcessingDone = new ManualResetEvent(false);
 
         private readonly ManualResetEvent _allOutputFinished = new ManualResetEvent(false);
-        
+
```

Comment:
This is caused by Productivity PowerTool, which will format the file when saving. Nothing changed but removed the spaces.